src/pytest_redis/factories.py

```diff
@@ -102,7 +108,11 @@ def redis_proc_fixture(request: FixtureRequest, tmpdir_factory: TempdirFactory):
         rdbcompression = config["compression"] if compression is None else compression
         rdbchecksum = config["rdbchecksum"] if checksum is None else checksum
 
-        tmpdir = tmpdir_factory.mktemp(f"pytest-redis-{request.fixturename}")
+        datadir = (
+            Path(unixsocketdir)
+            if unixsocketdir
```

Comment:
okay, this is not right at the moment. the parameter is unixsocketdir, yet it changes the location of logfiles, pidfile, and redis's working directory (other data files redis creates).

We should either have `unixsocketdir` as separate parameter also further down on RedisExecutor, OR better yet, simply name it datadir.

Apart from it, create a configuration option, command-line option update README with this parameter and then you won't have to configure it in code, or you'd actually have a choice how to change it.

Reply:
done, thanks!